packages/flutter/lib/src/services/raw_keyboard_common.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+
+/// Converts upper letters to lower letters in ASCII and extended ASCII, and
+/// returns as-is otherwise.
+///
+/// Independent of locale.
+int toLower(int n) {
```

Comment:
Does `String.toLowerCase()` not do extended ASCII? Otherwise, it would be preferable to use the Dart standard method.

They standard method appears to be pretty exhaustive:
https://github.com/dart-lang/sdk/blob/b06b60702b9e85585c3979c69ca91df85204173f/sdk/lib/_internal/vm/lib/string_patch.dart#L1214